src/lib/routes/admin-api/feature.js

```diff
@@ -39,7 +39,7 @@ class FeatureController extends Controller {
         this.post('/:featureName/toggle/off', this.toggleOff, UPDATE_FEATURE);
         this.post('/:featureName/stale/on', this.staleOn, UPDATE_FEATURE);
         this.post('/:featureName/stale/off', this.staleOff, UPDATE_FEATURE);
-        this.get('/:featureName/tags', this.listTags, UPDATE_FEATURE);
+        this.get('/:featureName/tags', this.listTags);
```

Comment:
@chriswk: It seemed like a bug that you must have the "UPDATE_FEATURE" rights in order to get tags for a feature toggle.

Reply:
Yes. Absolutely. Good catch